ice/mvc/app.zep

```diff
@@ -93,7 +93,8 @@ class App extends Access
                     view = controller->view;
 
                 // Load views and set the response body if auto render
-                if this->autoRender {
+                // and there is no content in the response
+                if this->autoRender && response->getBody() === null {
```

Comment:
Body is an empty string.
```zep
if this->autoRender && !response->getBody() {
```

Reply:
if user want to send empty string `$this->response->setContent('');`